src/app/App.svelte

```diff
@@ -1,50 +1,17 @@
-<script lang="ts">
-  import Upload from "./Upload.svelte";
-  import BCP47Tag from "./BCP47Tag.svelte";
-  export let name: string;
+<script>
+  import { Router, Route } from "svelte-routing";
+  import LandingPage from "./pages/LandingPage.svelte";
+  import Testing from "./pages/Testing.svelte";
 
-  let files: any;
+  // Used for SSR. A falsy value is ignored by the Router.
+  export let url = "";
 </script>
 
-<style>
-  :global(:root) {
-    /* Keyman brand colours, derived from https://keyman.com/ */
-    --blue: #6FBBD4;
-    --blue-light: #B3DBE7;
-    --red: #B90529;
-    --red-light: #DE929C;
-    --orange: #F18407;
-    --orange-light: #FBC88F;
-    --gray: #888888;
-    --gray-light: #D6D6D6;
-
-    /* Font stack */
-    /* Custom fonts downloaded from Google Fonts -- see public/index.html */
-    --main-font: "Cabin";
-    --mono-font: "Roboto Mono";
-  }
-
-  :global(body) {
-    font-family: var(--main-font), sans-serif;
-  }
-
-  :global(h1, h2, h3, h4, h5, h6) {
-    font-family: var(--main-font), sans-serif;
-  }
-
-  :global(pre, code, samp) {
-    font-family: var(--mono-font), monospace;
-  }
-</style>
```

Comment:
Could you actually keep these **global** styles here in `App.svelte`? These definitions should be cascaded to _all_ pages in the app 👍 

---

Honestly, figuring out where out global styles _should_ be is an issue in and of itself — I'm not entirely happy with it being in the `.svelte` files. But that can be handled in a different PR 😅  See also: #76.

Reply:
let me try creating a global css and see how things work so that we can keep the `App.svelte` as minimal as possible

Reply:
Yeah, honestly, getting the global styles out of `App.svelte` is the better option 👍